tst/exampleReadAsFunction.txt

```diff
@@ -0,0 +1,5 @@
+# This file contains a test input to read with ReadAsFunction
```

Comment:
Maybe put this in `tst/testinstall` instead of `tst`?

Reply:
Or just omit this, and only test `ReadAsFunction` for an input stream? They both go through `READ_AS_FUNC` in the end.

Reply:
I'd prefer to keep a test, just in the interests of the coverage. I'm happy to move the file.